src/dvclive/plots/utils.py

```diff
@@ -13,7 +12,7 @@
     "uint32",
     "uint64",
 ]
-NUMPY_FLOATS = ["float_", "float16", "float32", "float64"]
```

Comment:
Tests were failing due to numpy 2.0.

This solves this. And from what I can tell we don't need those types, they are not actual classes. so the check like:

`o.__class__.__name__ in NUMPY_INTS` (and the only way we are using it)

won't be successful.

cc @dberenbaum @skshetry